app/routes/inbox/regulatory-statements.js

```diff
@@ -0,0 +1,28 @@
+import Route from '@ember/routing/route';
+import { EDITOR_FOLDERS } from '../../config/constants';
+import { inject as service } from '@ember/service';
+
+export default class InboxRegulatoryStatementsRoute extends Route {
+  @service store;
+
+  queryParams = {
+    page: { refreshModel: true },
+    sort: { refreshModel: true },
+    filter: { refreshModel: true },
+  };
+
+  async model(params) {
+    const options = {
+      sort: params.sort,
```

Comment:
this view should sort on date created by default
https://binnenland.atlassian.net/browse/GN-3735 tracked here if too much work

Reply:
https://github.com/lblod/frontend-gelinkt-notuleren/pull/336